internal/pkg/template/workload.go

```diff
@@ -398,3 +416,24 @@ func envControllerParameters(o WorkloadOpts) []string {
 	}
 	return parameters
 }
+
+func (t Topic) ARN() string {
```

Comment:
Feel this is a little bit flaky as it won't work very well for other partition in future...Why don't we format the arn in the CFN template?

Reply:
@iamhopaul123 The ARN is being added to a JSON of ARNs so that the customer's env variables aren't spammed by a bunch of topics, since they can have many. Due to this it is being generated before adding to the CFN. Would you prefer I make this a function that takes a topic instead of a sub function of Topic?